build/Dockerfile.platform-validation

```diff
@@ -0,0 +1,23 @@
+FROM quay.io/sustainable_computing_io/kepler_builder:ubi-9-libbpf-1.2.0-go1.18 as builder
```

Comment:
this will build `validator` in a new image, can you add these to the builder and kepler docker files?

Reply:
validator is designed to be only used in platform-validation workflow which is executed ad-hoc or per release, do we need to build it regularly and maintain in official release?
And for debug use, developers also could build locally based on current Dockerfile to generate the standalone image to generate local raw node power data as comparison.